.circleci/deploy.sh

```diff
@@ -1,40 +1,28 @@
 #!/bin/bash
 set -eu -o pipefail
 
-SHA1=$1
-AWS_APPLICATION_NAME=$2
-export AWS_ENVIRONMENT_NAME=$3
-STATIC_BUCKET=$4
-export ENV_URL=$5
+SOURCE_BUNDLE=$CIRCLE_SHA1-config.zip
 
 function ebextensions_setup() {
     echo 'Setting up configuration for Papertrail logging'
-    export PAPERTRAIL_HOST=$(cut -d ":" -f 1 <<< $ENV_URL)
-    export PAPERTRAIL_PORT=$(cut -d ":" -f 2 <<< $ENV_URL)
     export PAPERTRAIL_SYSTEM=$AWS_ENVIRONMENT_NAME
     cat .ebextensions/03_papertrail.config | envsubst '$PAPERTRAIL_HOST:$PAPERTRAIL_PORT:$PAPERTRAIL_SYSTEM' >temp
     mv temp .ebextensions/03_papertrail.config
 
     echo 'Applying environment-specific configuration in .ebextensions'
     envsubst '$AWS_ENVIRONMENT_NAME' <.ebextensions/04_newrelic.config >temp
     mv temp .ebextensions/04_newrelic.config
-    envsubst '$ENV_URL' <.ebextensions/05_nginx_proxy.config >temp
+    envsubst '$APP_DOMAIN' <.ebextensions/05_nginx_proxy.config >temp
```

Comment:
@osahyoun if I'm not mistaken, in the nginx configuration we actually need the `$APP_DOMAIN` env var instead of `$ENV_URL`? 

`$ENV_URL` in this case was `log3.papertrailapp.com:<port>` (you can see it being split in host and port diff view above), which was argument `$5` (`$6` would have been the last arg we're passing, which is `actions.sumofus.org` in prod.

It also didn't seem like `$6` was being used anywhere.